ruby/introduction/installing_ruby.md

```diff
@@ -237,12 +254,21 @@ You'll notice nothing happened in the terminal. That's okay and is typical respo
 
 We can now (finally) install Ruby! Our curriculum currently uses version 3.1.2, which will allow you to complete this path's materials and content without error. We upgrade the material to accommodate newer versions as necessary. Without further ado, let's get going!
 
+First, let's upgrade `ruby-build`:
+
+~~~bash
+brew upgrade ruby-build
```

Comment:
Do you know what will happen here if the user doesn't have Ruby Build installed yet?

Reply:
I had brushed that possibility off as the user simply starting from Step 3.1 which includes the installation of `ruby-build`. I also notice that there is a sudden inconsistent jump from Step 3.1 to 3.3 without a Step 3.2, so we could add the `ruby-build` installation step there such that it's less miss-able as its own step?

Following that train of thought, I think it would be better to make that Step 3.1 and make the `rbenv` installation Step 3.2 and so that it's also not easily dismissable.

Reply:
Sounds good, give it a go and let's see how it looks :+1: